docs/application/dotnet/guides/nui/relative-layout.md

```diff
@@ -0,0 +1,86 @@
+# Relative layout
+
+The `RelativeLayout` is a layout class in which each side of children can be specified as relative to a sibling view or parent. The `Target` and the `RelativeOffset` properties define a relationship with each other.
```

Comment:
would it be more accurate to say "a child" than children?
Please see my suggestion below:
The `RelativeLayout` is a layout class in which each side of a child can be specified as relative to a sibling view or parent.

Reply:
@annie-samsung It would have been good, but there is more than one child.